olexandr.vakulinskiy/HW3/Task1/prime_numbers.rb

```diff
@@ -0,0 +1,54 @@
+require 'matrix'
+class Matrix
+  def print_matrix
+    count = 0
+    each do |number|
+      if count < column_size - 1
+        print number.to_s + ', '
+        count += 1
+      else
+        print number.to_s + ' '
+        count += 1
+      end
+      if count == column_size
+        print "\n"
+        count = 0
+      end
```

Comment:
@AlexanderVak Please, change these conditions to `case`. It will be much more readable.

Reply:
How to set less than case in when?